pkg/build/strategies/sti/sti_test.go

```diff
@@ -149,17 +149,17 @@ func TestBuild(t *testing.T) {
 			t.Errorf("Unexpected optional scripts requested: %#v", fh.SetupOptional)
 		}
 
-		// Verify that determineIncremental was called
+		// Verify that Determine was called
 		if !fh.DetermineIncrementalCalled {
 			t.Errorf("Determine incremental was not called.")
 		}
 
-		// Verify that saveartifacts was called for an incremental build
+		// Verify that Save was called for an incremental build
```

Comment:
I based the changes in the fact that the interface for [`IncrementalBuilder`](https://github.com/openshift/source-to-image/blob/d2c8661cf024ad774210a218b03717b2673d53e1/pkg/build/interfaces.go#L29-L32) is:
```go
type IncrementalBuilder interface {
	Determine(*api.Request) error
	Save(*api.Request) error
}
```